api/src/main/java/org/pmiops/workbench/shibboleth/ShibbolethRetryHandler.java

```diff
@@ -0,0 +1,60 @@
+package org.pmiops.workbench.shibboleth;
```

Comment:
I basically copied this from FirecloudRetryHandler. It felt like an awful lot of boilerplate for a single new API call, but I didn't see an obvious alternative.